### DISCUSSION TITLE
ChannelVideo传参错误

### DISCUSSION THREAD

**Discussion by tingting-lucy** [NervJS/taro#16219] [Q & A]
ChannelVideo组件报错  Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.[missing argument]

<ChannelVideo
                    feedId={'export/UzFfAgtgekIEAQAAAAAANq81OZexBwAAAAstQy6ubaLX4KHWvLEZgBPEh4JEVQFjJcqLzNPgMJpyE5t48bzAc-v1l1ug3tId'}
                    finderUserName={'sphz8MMayKe3fca'}
                    feedToken={'token/ACB5C55M8'}
                    autoplay={false}
                  ></ChannelVideo>

**Comment by tingting-lucy** [comment]
不知道问题在哪，有没有人成功过

**Reply by honlyHuang** [reply]
你提供的信息太少了，去提个issue ?

**Reply by tingting-lucy** [reply]
提了的，但是一直没有人回应
